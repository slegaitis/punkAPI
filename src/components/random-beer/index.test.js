import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import RandomBeer from './';

import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';

const mockStore = configureMockStore();
const store = mockStore({});

test('Random Beer component should render', () => {
	const component = shallow(
		<Provider store={store}>
			<RandomBeer />
		</Provider>
	);
	const tree = toJson(component);

	expect(tree).toMatchSnapshot();
});
