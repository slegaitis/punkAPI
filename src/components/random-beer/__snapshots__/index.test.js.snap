// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Random Beer component should render 1`] = `<Connect(RandomBeer) />`;
